### ISSUE TITLE
Fix or silence "cannot find fiber with ID" warnings when hovering

### ISSUE THREAD

**Issue by gaearon** [bvaughn/react-devtools-experimental#166]
Sometimes I see "cannot find fiber with ID" when hovering over the tree. They are sporadic. I'll need to dig into why they happen. But we also need to look at our usage of warnings to make sure they're either actionable or gated by DEV/DEBUG.

**Comment by bvaughn** [comment]
I agree this is probably not ideal currently. What are "expected" edge cases and what are legit errors? I've tried to replace legit error cases with `throw` statements recently, but I understand that it's still not the clearest.

Some (most?) of the the remaining warnings are expected potential edge cases. A little more context...

For example, if an element is selected in the DevTools frontend, then it gets unmounted on the page/backend, we may temporarily see this warning when we query for its inspected props:
https://github.com/bvaughn/react-devtools-experimental/blob/20193f03c04971a14ee8a0b6755bdfee08b94ba3/src/backend/renderer.js#L1432-L1438

Similarly, since the `Store` is mutable, it's possible that a node will be removed while a render batch is in progress:
https://github.com/bvaughn/react-devtools-experimental/blob/85179fe4a7e7cb2e9d79733eec4839443393e7ba/src/devtools/views/Components/Element.js#L122-L128

**Comment by bvaughn** [comment]
This repository is being merged into the main React repo ([github.com/facebook/react](https://github.com/facebook/react/)). As part of this, I am moving all issues to that repository as well and closing them here.

This issue has been relocated to:
https://github.com/facebook/react/issues/16493
